cli/Sources/TuistAutomation/XcodeBuild/XcodeBuildController.swift

```diff
@@ -341,15 +342,20 @@ public final class XcodeBuildController: XcodeBuildControlling {
         let command = ["/usr/bin/xcrun", "xcodebuild"] + arguments
         
         logger.debug("Running xcodebuild command: \(command.joined(separator: " "))")
+       
+        var output = Data()
```

Comment:
The formatting of the output happens as blobs fo data come through stdout and stderr using [formatter](https://github.com/ns-vasilev/tuist/blob/df64aa36d2d774d7560ea7968a8a13ad12d16ceb/cli/Sources/TuistAutomation/XcodeBuild/XcodeBuildController.swift#L20). What you want is configuring the formatter with options. Note that in our case we are not shelling out to `xcbeautify`, which means we'd need to duplicate the parsing of arguments to runtime options.

@cpisciotta would you consider having the arguments and options defined in an exported product that we can consume from here and that you can also reuse in the `xcbeautify` CLI?